### PR TITLE
add function convertIRF

### PR DESCRIPTION
**Pull request by Fsioutas**

### CODE REVIEW

File: R/convertIRF.R [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#' convertIRF
+#'
+#' Add missing periods to data frame and interpolate missing values.
+#' The ISO codes of "IRF" data are compared with the official ISO code country list.
+#' NA values are replaced with zeros and historical data are corrected.
```
Comment (giannou) [inline R/convertIRF.R:5]: historical data are not corrected - there's no call of `toolISOhistorical` here

File: R/convertIRF.R [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#' \dontrun{
+#' a <- readSource("IRF", convert = TRUE)
+#' }
+#'
+#' @importFrom quitte as.quitte
+#' @importFrom quitte interpolate_missing_periods
+
+convertIRF <- function(x) {
+
+  x <- as.quitte(x) %>%
```
Comment (giannou) [inline R/convertIRF.R:23]: shouldn't `%>%` be included in the function documentation?

File: R/convertIRF.R [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#'
+#' @author Anastasis Giannousakis, Fotis Sioutas
+#'
+#' @examples
+#' \dontrun{
+#' a <- readSource("IRF", convert = TRUE)
+#' }
+#'
+#' @importFrom quitte as.quitte
+#' @importFrom quitte interpolate_missing_periods
```
Comment (giannou) [inline R/convertIRF.R:19]: no need for a second line for the quitte imports, they can all be in the same line separated by spaces

**Review by giannou — CHANGES_REQUESTED** [review]

File: R/convertIRF.R (hunk 1 above)
Comment (Fsioutas) [reply R/convertIRF.R]: ok

File: R/convertIRF.R (hunk 2 above)
Comment (Fsioutas) [reply R/convertIRF.R:23]: ok

File: R/convertIRF.R (hunk 3 above)
Comment (Fsioutas) [reply R/convertIRF.R]: ok

**Review by giannou — APPROVED** [review]

**Pull request merged**